src/main/java/io/nats/client/impl/NatsConnection.java

```diff
@@ -1829,7 +1829,7 @@ void queueOutgoing(NatsMessage msg) {
         }
     }
 
-    void queueInternalOutgoing(NatsMessage msg) {
+    void queueInternalOutgoing(ProtocolMessage msg) {

```

Comment:
All internal messages were ProtocolMessages, so I just changed the type to reflect this.